GoogleForCreators/web-stories-wp
# Reduce size of template images by 3/4

## Context

Template images are too big and affecting editor load performance. See #4768.

## Summary

Resizes template images to either 1080p (full width), 720p (large images), or 480p (small images). 

Before: 
```
$ du .
18632	./cooking
23968	./diy
17616	./entertainment
46664	./fashion
54688	./beauty
42744	./fitness
27256	./travel
18872	./wellbeing
250464	.
```

After:
```
$ du .
5384	./cooking
4032	./diy
3208	./entertainment
5840	./fashion
5048	./beauty
37992	./fitness
4488	./travel
3408	./wellbeing
69424	.
```

## Relevant Technical Choices

N/A

## To-do

- [ ] Update editor code to reference new JPG variants of existing PNGs
- [ ] Increase `max-age` of images to 30d or longer
- [ ] Update template creation docs with asset sizing guidance
- [ ] (Optional) Add pre-commit hook that sets a maximum image resolution size

## User-facing changes

Should be transparent to users.

## Testing Instructions



### QA



1. Pick a few templates to create new stories from
2. Verify that images in the templates still look good

### UAT




## Reviews

### Does this PR have a security-related impact?

No.

### Does this PR change what data or activity we track or use?

No.

### Does this PR have a legal-related impact?

No.

## Checklist




---



Fixes #4768
